src/main/java/aztech/modern_industrialization/machines/multiblocks/MultiblockMachineBlockEntity.java

```diff
@@ -45,7 +47,61 @@ public boolean isShapeValid() {
         return shapeValid.shapeValid;
     }
 
-    public abstract void unlink();
+    public ShapeMatcher getShapeMatcher() {
```

Comment:
Current MI doesn't need this. Do you?

Reply:
I use it to check if my ShapeMatcher (SameCableTierShapeMatcher) has mismatching hatches or not and display info about it in the multiblock GUI (see [here](https://github.com/Swedz/Extended-Industrialization/blob/6b730612783d5d0dd8376044dc9bd1e88056bd16/src/main/java/net/swedz/extended_industrialization/machines/blockentity/multiblock/teslatower/TeslaTowerBlockEntity.java#L128-L131)). Mismatching hatches in this case refers to if there are energy hatches that aren't the same cable tier as others.

Reply:
I'd probably change that info to be stored as a field in the BE, updated in a callback. I don't want the shape matcher to be too accessible.